app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,13 +3,10 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  role: null,
+  tags: null,
```

Comment:
Couldn't we make the default value for `tags` as empty array `[]`? Wouldn't that be more appropriate from a business perspective?

Currently let's say we add two tags. Then we remove both of them. Then the value will be an empty array and not a `null` right?